contribs/gmf/examples/datepicker.js

```diff
@@ -2,23 +2,26 @@ goog.provide('gmfapp.datepicker');
 
 /** @suppress {extraRequire} */
 goog.require('ngeo.DatePickerDirective');
-goog.require('ngeo.WMSTime');
+goog.require('ngeo.misc.WMSTime');
 
 
 /** @type {!angular.Module} **/
-gmfapp.module = angular.module('gmfapp', ['gmf']);
+gmfapp.module = angular.module('gmfapp', [
```

Comment:
This example doesn't work any more for me. I think we need to require the gmf module.